tools/release/create-draft-release-notes.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -e
+
+# Load common lib.
+CURR_DIR="$(dirname "$0")"
+. "${CURR_DIR}/common.sh"
+
+CHANGELOG_PATH="${CURR_DIR}/../../CHANGELOG.md"
+
+#
+# Contributors
+#
+
+NUM_PRS=$(git log --pretty=format:"%s" "${LAST_RELEASE_TAG}...${PREV_RELEASE_TAG}" | grep -Eo '#[0-9]+' | wc -l | grep -Eo '[0-9]+')
```

Comment:
`$LAST_RELEASE_TAG` isn't defined until you call `find_last_release`, right?

Can you also remind me of why you use the three-dot syntax instead of two-dot?

Reply:
Sorry for that. It was a mistake I introduced during a final refactoring, after the release was published. Fixed and opened a new PR here: https://github.com/grafana/mimir/pull/3187